### PR TITLE
Nuke behavior more resilient

### PR DESCRIPTION
**Pull request by jrsndl**
## Changelog Description
Ayon uses onScriptSave callback to update nodes' colors. If this fails by any reason, artist is not able to save the file.

Nuke sometimes fails on getting knob type needed in ```read_avalon_data```. This can result in artisn not being able to open workfile. Reason unknown, hard to test.

Changes allow artist to continue working.

## Additional info
Might close https://github.com/ynput/ayon-nuke/issues/33

## Testing notes:
1. Launch Nuke, open Workfile
2. Make changes
3. Stop server
4. Save file with Control + S

### CODE REVIEW

File: client/ayon_nuke/api/lib.py [hunk 1]
```diff
@@ -593,10 +593,13 @@ def compat_prefixed(knob_name):
...
-            knob_type = nuke.knob(knob.fullyQualifiedName(), type=True)
-            value = knob.value()
-
+            try:
+                knob_type = nuke.knob(knob.fullyQualifiedName(), type=True)
+                value = knob.value()
+            except Exception:
+                log.debug(
+                    f"Error in knob {knob_name}, node {node['name'].value()}")
+                continue
```
Comment (jakubjezek001) [inline client/ayon_nuke/api/lib.py:602]: This is a bit of a puzzle to be honest. I am not sure how this was blocking the processing. The code is just getting knob type and value. This seems to be redundant.

Comment (jrsndl) [reply client/ayon_nuke/api/lib.py:602]: Nuke sometimes breaks on the knob type line. This can block opening Nuke scripts.

Comment (jrsndl) [reply client/ayon_nuke/api/lib.py:602]: Unfortunatelly, I can't find any examples stached for this; it only happens few times in few thousands shots.

**Review by jakubjezek001 — APPROVED** [review]
looking good

**Pull request merged**